matchms/calculate_scores.py

```diff
@@ -5,7 +5,6 @@
 
 def calculate_scores(references: ReferencesType, queries: QueriesType,
```

Comment:
I think we should not have a `calculate_scores()` function AND a `BaseSimilarity.calculate_scores()` method with different implementations. Could we not have a function `calculate_scores()` which is then used in the BaseSimilarity class?

Reply:
Good point. I have removed calculate_scores as a method and converted it to a function. I renamed the original calculate_scores function to create_scores_object_and_calculate_scores. I am fine with removing this in the end, but it was currently used in many tests, so I kept it for now.